aws/resource_aws_lambda_event_source_mapping.go

```diff
@@ -196,7 +213,9 @@ func resourceAwsLambdaEventSourceMappingUpdate(d *schema.ResourceData, meta inte
 		_, err := conn.UpdateEventSourceMapping(params)
 		if err != nil {
 			if awserr, ok := err.(awserr.Error); ok {
-				if awserr.Code() == "InvalidParameterValueException" {
+				if awserr.Code() == "InvalidParameterValueException" ||
```

Comment:
Minor nitpick: we have a helper function available for this type of logic and also prefer to use the SDK available constants, e.g.

```go
if isAWSErr(err, lambda.ErrCodeInvalidParameterValueException, "") || isAWSErr(err, lambda.ErrCodeResourceInUseException, "") {
```

Reply:
Fixed.